lang/test/org/partiql/lang/eval/evaluatortestframework/AstEvaluatorTestAdapterTests.kt

```diff
@@ -0,0 +1,417 @@
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS mode (bag)`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "<<1>>",
+                    // note: In this ExpectedResultFormat we lose the fact that this a BAG and not an
+                    // ordinary Ion list. This is unfortunate, but expected.
+                    expectedResult = "[1]",
+                    expectedResultFormat = ExpectedResultFormat.ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS mode (missing)`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "MISSING",
+                    // note: In this ExpectedResultFormat we lose the fact that this MISSING and not an
+                    // ordinary Ion null. This is unfortunate, but expected.
+                    expectedResult = "null",
+                    expectedResultFormat = ExpectedResultFormat.ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS mode (date)`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "DATE '2001-01-01'",
+                    expectedResult = "\$partiql_date::2001-01-01",
+                    expectedResultFormat = ExpectedResultFormat.ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS mode (time)`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "TIME '12:12:01'",
+                    expectedResult = "\$partiql_time::{hour:12,minute:12,second:1.,timezone_hour:null.int,timezone_minute:null.int}",
+                    expectedResultFormat = ExpectedResultFormat.ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-PARTIQL mode`() {
```

Comment:
This test is run in `ExpectedResultFormat.ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS`?

Reply:
Corrected the name.